pdns/pdnsutil.cc

```diff
@@ -331,7 +331,7 @@ int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, const vect
     }
     catch(std::exception& e)
     {
-      cout<<"[Error] Following record had a problem: "<<rr.qname<<" IN " <<rr.qtype.getName()<< " " << rr.content<<endl;
+      cout<<"[Error] Following record had a problem: \""<<rr.qname<<" IN " <<rr.qtype.getName()<< " " << rr.content "\"" <<endl;
```

Comment:
Missing `<<` between `rr.content` and `"\""`.

Reply:
thanks